common/cpp/crypto/verify_signature.cpp

```diff
@@ -54,7 +54,7 @@ bool verify_signature(const char* cert_pem,
     unsigned char signature_decoded[signature_decoded_len + 3];
     ret = EVP_DecodeBlock(signature_decoded,
                           (unsigned char*)signature,
-                          signature_len + 3);
+                          signature_len);
```

Comment:
Can you remove this file from commit as this is not documentation fix?

Reply:
That was  rebase error. That fix is already in the code.  I fixed it--please re-review.